docs/devcontainer-spec-support.md

```diff
@@ -0,0 +1,72 @@
+| 🔴                            | `userEnvProbe`                        | Shell to use when probing for user environment variables.                                                              | -                                                      |
+| 🔴                            | `overrideCommand`                     | Override the default sleep command to be run by supporting services.                                                   | -                                                      |
+| 🔴                            | `shutdownAction`                      | Action to take when supporting tools are closed or shut down.                                                          | -                                                      |
+| 🔴                            | `init`                                | Adds a tiny init process to the container.                                                                             | [#221](https://github.com/coder/envbuilder/issues/221) |
+| 🔴                            | `privileged`                          | Whether the container should be run in privileged mode.                                                                | -                                                      |
+| 🔴                            | `capAdd`                              | Capabilities to add to the container (for example, `SYS_PTRACE`).                                                      | -                                                      |
+| 🔴                            | `securityOpt`                         | Security options to add to the container (for example, `seccomp=unconfined`).                                          | -                                                      |
+| 🔴                            | `mounts`                              | Add additional mounts to the container.                                                                                | [#220](https://github.com/coder/envbuilder/issues/220) |
+| 🟢                            | `features`                            | Features to be added to the devcontainer.                                                                              | -                                                      |
+| 🔴                            | `overrideFeatureInstallOrder`         | Override the order in which features should be installed.                                                              | [#226](https://github.com/coder/envbuilder/issues/226) |
+| 🔴                            | `customizations`                      | Product-specific properties, e.g., _VS Code_ settings and extensions.                                                  | [#43](https://github.com/coder/envbuilder/issues/43)   |
+| **Image/Dockerfile**          |                                       |                                                                                                                        |
```

Comment:
Might be clearer to split these into separate tables instead of "sections".

Reply:
Can do!